tl-services/src/main/resources/application.properties

```diff
@@ -10,14 +12,14 @@ spring.datasource.password=postgres
 
 ##----------------------------- FLYWAY CONFIGURATIONS ------------------------------#
 
-flyway.url=jdbc:postgresql://localhost:5432/rainmaker_tl
-flyway.user=postgres
-flyway.password=postgres
-flyway.table=public
-flyway.baseline-on-migrate=true
-flyway.outOfOrder=true
-flyway.locations=db/migration/main
-flyway.enabled=true
+spring.flyway.url=jdbc:postgresql://localhost:5432/rainmaker_tl
+spring.flyway.user=postgres
+spring.flyway.password=postgres
+spring.flyway.table=public
+spring.flyway.baseline-on-migrate=true
+spring.flyway.outOfOrder=true
+spring.flyway.locations=classpath:/db/migration/main
```

Comment:
these changes are already in master, how come the branch you are merging it to doesn't have it?